aidants_connect_carto_api/models.py

```diff
@@ -105,6 +105,13 @@ class Place(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    def __str__(self):
+        return f"{self.name}"
+    
+    @property
+    def service_count(self):
+        return Service.objects.filter(place=self.id).count()
```

Comment:
Tu peux écrire ça plus simplement en partant de `self` :
```suggestion
        return self.service_set.count()
```

Voire rajouter explicitement un `related_name` sur la définition du champ qui porte la relation pour pouvoir écrire un truc plus sympa genre

```python
        return self.services.count()
```

Reply:
en effet, il a d'ailleurs un related_name donc ta dernière ligne marche très bien

Reply:
(perso je n'aime pas [la nomenclature `BIDULE_set` utilisée par défaut par Django](https://docs.djangoproject.com/en/3.0/topics/db/queries/#following-relationships-backward) donc j'ajoute systématiquement des `related_name` à tous mes champs `ForeignKey`